staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -43,7 +43,7 @@ import (
 // desirable because it means that CEL expressions are portable across a wider range
 // of Kubernetes versions.
 func DefaultCompatibilityVersion() *version.Version {
-	return version.MajorMinor(1, 28)
+	return version.MajorMinor(1, 29)
```

Comment:
Remove this from this PR since it's testing capabilities added in 1.28?

Reply:
I also added test in validation_test which require the version bump to be in place